cli/docs/cli.json

```diff
@@ -146,6 +146,26 @@
             }
           ]
         },
+        {
+          "name": "rename-profile",
+          "about": "Rename a profile.",
+          "long_about": "Rename a profile.\n\nThis command does not modify the profile credentials. If\nthe profile being renamed is your `default-profile` that\nwill be updated as well.",
+          "args": [
+            {
+              "long": "profile",
+              "help": "Configuration profile to use for commands",
+              "global": true
+            },
+            {
+              "name": "current_profile_name",
+              "help": "The current name of the profile to be renamed"
+            },
+            {
+              "name": "new_profile_name",
```

Comment:
I think these are supposed to be kebab case

Reply:
> I also think “profile” is redundant, could be “new name”. 

Good call, I think this is clearer as just `new_name`. Updated in e70579e

> I think these are supposed to be kebab case.

Long flags should be kebab, but the value passed is always SCREAMING_SNAKE_CASE:

```
  -f, --raw-field <RAW_FIELD>
```

Reply:
Oh right, it's a positional args. I had a thought yesterday that maybe we should be more explicit in the JSON about what kind of arg it is, like adding `positional: true` even though it's redundant. I think I'll get used to it, though.

On the other hand, maybe it would be nice if these weren't positional? I know no amount of documentation can make me remember the order positional args are supposed to go in.

Reply:
> maybe it would be nice if these weren't positional?

Personally I think the positional args are intuitive in this case, but I would, wouldn't I? Purely flags driven UIs like [firewall-cmd](https://firewalld.org/documentation/man-pages/firewall-cmd.html) drive me up the wall, but I can't deny that they're self-documenting.